### ISSUE TITLE
Timeline is acting weird.

### ISSUE THREAD

**Issue by Loink1** [worldanvil/worldanvil-bug-tracker#978]
**World Anvil Username**:
**Loink**

**World / Campaign / RPG Character / Manuscript Title**:

**Feature**: 

**Related URLS** (To public & visible pages add screenshots otherwise):
**https://www.worldanvil.com/w/minecraftia-loink/t/minecraft-s-development-cycle-timeline?preview=1#timeline-start**

**Describe the Issue**
A clear and concise description of what the issue is. 
**I hope to show other people the timeline of my story. So I began testing the article with another timeline and trying to see if I could see it in incognito mode, since I'm not logged in while incognito. But it doesn't show any of the content I put in the timeline outside of the very first era. The prompt to click on 'Go to the beginning' causes the screen to move down, but does nothing else.**

**To Reproduce**
**To reproduce this, you'll probably need to make a new timeline by going to the section on your dashboard that says 'Timeline', clicking 'Create a new Timeline'(make sure to add text so the site knows you actually intended to make a timeline), then adding at least two different historical entries. After that, go ahead and View your timeline. At least from my experience, whether or not you're signed in should not make a difference.**

**Expected behavior**
**What should happen is that your timeline will display only one historical entry with clickable text saying 'Go to the Beginning', which if clicked on will move your screen down.**

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!
![Screenshot 2022-04-12 052117](https://user-images.githubusercontent.com/103494999/162926920-d5bd8157-5305-4576-95ff-c5819abe9089.png)

**Comment by Kordishal** [comment]
This is not a bug. Your Era `Creation` goes from year 0 to 0. This means it does not hold any Events as they can't happen in no time.

The events are probably only displayed under the OPRHAN HISTORIES banner which indicates that they do not have an ERA that contains them. These events are only visible to the author. Which is why you cannot see them if you are not logged in.

**Comment by Loink1** [comment]
The issue is, even when I remove the Orphan Histories and add end dates to the events, I still cannot see them.

**Comment by Kordishal** [comment]
You need ERAs for your events to go into. Not sure how you are removing orphan histories, that isn't really a thing.

**Comment by Loink1** [comment]
I agree it would probably be better to meet up on Discord to solve this issue. My Discord username is Loink;%

**Comment by Kordishal** [comment]
Please join the World Anvil Server: https://worldanvil.com/discord and ask in the #help channel for help :)
